### PR TITLE
chore: Initial project setup

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
Initial Typescript setup with eslint and jest testing.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@asyncapi/generator-model-sdk",
+  "version": "0.1.1",
```
Comment (magicmatatjahu) [inline package.json:3]: 0.1.0 :P

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (magicmatatjahu) [inline LICENSE:189]: 2021 AsyncAPI org?

Comment (jonaslagoni) [reply LICENSE:189]: hmm, @derberg should LICENSE file not be pushed from .github repository?

Comment (derberg) [reply LICENSE]: don't worry, it is the same everywhere, even generator 😄 

LICENSE file is not global. I assume you ask in the context of other files we have there, like security or contributing

**Review by magicmatatjahu — APPROVED** [review]

**Pull request merged**